pkgs/applications/system/glances/default.nix

```diff
@@ -44,24 +43,28 @@ buildPythonPackage rec {
   ];
 
   doCheck = true;
-  checkInputs = [ unittest2 ];
   preCheck = lib.optional stdenv.isDarwin ''
     export DYLD_FRAMEWORK_PATH=/System/Library/Frameworks
   '';
 
-  propagatedBuildInputs = [ psutil setuptools bottle batinfo pysnmp future
+  propagatedBuildInputs = [
+    batinfo
+    bottle
+    future
     netifaces
+    psutil
+    pysnmp
+    setuptools
   ] ++ lib.optional stdenv.isLinux hddtemp;
 
   preConfigure = ''
     sed -i 's/data_files\.append((conf_path/data_files.append(("etc\/glances"/' setup.py;
   '';
 
   meta = with lib; {
-    homepage = https://nicolargo.github.io/glances/;
+    homepage = "https://nicolargo.github.io/glances/";
     description = "Cross-platform curses-based monitoring tool";
     license = licenses.lgpl3;
-    maintainers = with maintainers; [ primeos koral ];
-    platforms = platforms.linux ++ platforms.darwin;
+    maintainers = with maintainers; [ jonringer primeos koral ];
```

Comment:
@jonringer just to clarify: Will you take care of the updates then? If so you can probably remove @k0ral and me. (If not I could of course keep maintaining it)

Reply:
I usually check repology every month to see if i need to update anything. I don't think it hurts having many maintainers

Reply:
I would be glad to be kept in the maintainers list, as far as I'm concerned :) .

Reply:
> I usually check repology every month to see if i need to update anything. I don't think it hurts having many maintainers

ok, in that case I'll keep following the upstream release feed for new updates